exporter/signalfxexporter/exporter_test.go

```diff
@@ -1074,8 +1074,8 @@ func TestConsumeMetadata(t *testing.T) {
 
 			select {
 			case <-c:
-			// wait 5ms longer than send delay
-			case <-time.After(time.Duration(tt.sendDelay)*time.Second + 5*time.Millisecond):
+			// wait 500ms longer than send delay
```

Comment:
Maybe `100ms` is enough? It's +0.5 second to every test run

Reply:
I considered 50ms but if it happens again we're pushing more time obligations w/ reruns to contrib devs and would have to bump it again (unless a better fix is apparent). To be fair I think each test used to be 1s until the previous updates. Ok w/ 100ms if you think it's still worth it.

Reply:
To clarify only one test uses the timeout unless bugs have creeped in:

```
=== RUN   TestConsumeMetadataNotStarted
--- PASS: TestConsumeMetadataNotStarted (0.00s)
=== RUN   TestConsumeMetadata
=== RUN   TestConsumeMetadata/Test_property_updates
=== RUN   TestConsumeMetadata/Test_tag_updates
=== RUN   TestConsumeMetadata/Test_quick_successive_updates
=== RUN   TestConsumeMetadata/Test_updates_on_dimensions_with_nonalphanumeric_characters_(other_than_the_default_allow_list)
=== RUN   TestConsumeMetadata/no_dimension_update_for_empty_properties
--- PASS: TestConsumeMetadata (1.51s)
    --- PASS: TestConsumeMetadata/Test_property_updates (0.00s)
    --- PASS: TestConsumeMetadata/Test_tag_updates (0.00s)
    --- PASS: TestConsumeMetadata/Test_quick_successive_updates (1.00s)
    --- PASS: TestConsumeMetadata/Test_updates_on_dimensions_with_nonalphanumeric_characters_(other_than_the_default_allow_list) (0.00s)
    --- PASS: TestConsumeMetadata/no_dimension_update_for_empty_properties (0.50s)
```

Reply:
Ok, sounds good. let's keep 500ms given that you made it faster last time :)